src/LanguageServer/Impl/LanguageServer.Configuration.cs

```diff
@@ -162,5 +171,29 @@ public sealed partial class LanguageServer {
 
             return ImmutableArray<string>.Empty;
         }
+
+        private ImmutableArray<string> AutoSearchPathHeuristic() {
```

Comment:
Do you want to add tests for it?

Reply:
Sure, I can see about moving this to another place to test directly given a `IFilesystem`.